enterprise/reporting/reporting.go

```diff
@@ -84,6 +85,7 @@ func NewDefaultReporter(ctx context.Context, log logger.Logger, configSubscriber
 	maxConcurrentRequests := config.GetReloadableIntVar(32, 1, "Reporting.maxConcurrentRequests")
 	maxOpenConnections := config.GetIntVar(32, 1, "Reporting.maxOpenConnections")
 	dbQueryTimeout = config.GetReloadableDurationVar(60, time.Second, "Reporting.dbQueryTimeout")
+	autoVacuumCostLimit := config.GetIntVar(2000, 1, "Reporting.autoVacuumCostLimit")
```

Comment:
We can keep the default value to 200
```suggestion
	autoVacuumCostLimit := config.GetIntVar(200, 1, "Reporting.autoVacuumCostLimit")
```

Reply:
Done